app/src/main/res/values/strings.xml

```diff
@@ -964,6 +964,9 @@
     !-- This is the title of promote search widget dialog. -->
     <string name="promote_search_widget_dialog_title">Browsing history cleared! 🎉</string>
 
+    <!-- CFR for Start your private browsing session-->
```

Comment:
Generally it is preferred to avoid acronyms. Maybe localizers don't know these internal namings.
Would be useful to reformulate this to avoid the `CFR` or at least offer more context. I see for other such strings that the comment contains `(Banner Info Message)`.